### ISSUE TITLE
typescript

### ISSUE THREAD

**Issue by createthis** [carlhuda/janus#666]
Hey, can we get some typescript syntax highlighting up in this piece? Angular 2.x and all that.

For anyone looking for a quick fix, this seems to work fine:

```
mkdir ~/.janus

cd ~/.janus

git clone https://github.com/leafgarland/typescript-vim.git typescript-vim
```

Now any .ts you open will have syntax highlighting.

Coolest issue number ever BTW. I win!

**Comment by aaaronic** [comment]
👍

**Comment by kalbasit** [comment]
feel free to open a PR with the added TS plugin

**Comment by createthis** [comment]
Do you have a howto guide for building/testing janus during development? I'm guessing I just need to do something like this? https://github.com/carlhuda/janus/pull/273/commits/7d37679a6e0c5001bd5a85d5d1fd199e9b1b60e3

**Comment by kalbasit** [comment]
I don't unfortunately but yes that's exactly what you need, also please add it to https://github.com/carlhuda/janus/blob/master/janus/submodules.yaml

**Comment by chauncey-garrett** [comment]
This can be closed via #667.
